app/views/cohorts/edit.haml

```diff
@@ -37,7 +37,7 @@
       = f.input :viewer_ids, collection: @users, as: :select_two, selected: @viewer_ids, input_html: {multiple: :multiple, style: 'width: 100%;'}, label: 'Grant read-only-level access to users', hint: 'Specified user will have access to view, but not change clients on this cohort.'
       %h3 Automation
       .well
-        = f.input :project_group_id, collection: GrdaWarehouse::ProjectGroup.options_for_select(user: current_user), as: :select_two, include_blank: 'Manually controlled (no project group)', hint: 'If you select a project group, this cohort will always contain clients with an open enrollment at a project contained in the group.  You will not be able to add and remove clients manually.'
+        = f.input :project_group_id, disabled: !@cohort.selected_project_group_viewable_by(current_user), collection: @cohort.project_group_options_for_select(current_user), as: :select_two, include_blank: 'Manually controlled (no project group)', hint: 'If you select a project group, this cohort will always contain clients with an open enrollment at a project contained in the group.  You will not be able to add and remove clients manually.'
```

Comment:
@dtgreiner in your digging, do you know if the controller checks visibility of the project group on update?  I'm tempted to leave this editable, but ensure that the user can always safely submit the existing project group.  That way they can change the project group if necessary, but it won't remove it if they don't have access.  I'd expect something in the controller that checks to see if they can see the project group OR if the project group is on the cohort pre-save.

Reply:
Nothing popping out about project_groups pre-save but it looks like adding it in would only be a couple lines to update. If I'm reading this right, we want the following:

1. show the current selection even if the current user doesn't have access to view the group
2. leave the selection as editable
3. validate selection pre-save on the controller to ensure the new selection is either Viewable by the current user OR existed as the project group prior to save

This would let the user change the project group or remove it even if they don't have access to view the current selection, and it would prevent overwriting the group inadvertently like this scenario.

Reply:
@eanders I just pushed a new commit that should address this

Reply:
Thank you, and yes, that's the spec I was shooting for.